main/webapp/modules/core/externals/jquery-ui/css/ui-lightness/theme.css

```diff
@@ -172,20 +172,14 @@ a.ui-button:active,
 .ui-widget-content .ui-priority-secondary,
 .ui-widget-header .ui-priority-secondary {
 	opacity: .7;
-	-ms-filter: "alpha(opacity=70)"; /* support: IE8 */
```

Comment:
This seems to be an external library that we have copied into the repository, I would rather not make changes to it here. If there is a newer version of it we could upgrade to it.

Reply:
Hi,

JQuery UI is generated from the website https://jqueryui.com/download/ , so we cannot add it to the package.json. We are already using the latest version (1.13.1.), so I don't think we should modify those files unless it is causing an issue.

Reply:
A mistake on my side, I did not intend to modify external dependencies.